dev/tests/integration/framework/Magento/TestFramework/TestCase/AbstractController.php

```diff
@@ -15,6 +15,7 @@
 
 /**
  * @SuppressWarnings(PHPMD.NumberOfChildren)
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
But at the same time you have decoupled this class from `\Zend_Json` and `\Zend_Json_Exception`. So to me it looks like that same coupling value should be in place after your changes. 
Am I missing something?